### PR TITLE
Change sinon lib

### PR DESCRIPTION
**Pull request by benrudolph**
Think this is causing travis to fail

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   "devDependencies": {
     "chai": "3.3.0",
     "mocha": "2.3.3",
-    "sinon": "http://sinonjs.org/releases/sinon-1.17.0.js",
+    "sinon": "http://sinonjs.org/releases/sinon-1.17.7.js",
```
Comment (emord) [inline bower.json]: Why do these use bower.json versus our package.json https://github.com/dimagi/commcare-hq/blob/master/package.json ? seems like our test dependencies are split up

Comment (benrudolph) [inline bower.json]: server vs. browser js libs. sinon gets loaded into the browser where as npm is for server js like cli calls to the js-xpath module

Comment (benrudolph) [inline bower.json]: the description is mighty confusing though

Comment (emord) [inline bower.json]: Ah. That makes sense. Thanks

**Pull request merged**